Admin-app/src/components/SidebarItems/index.js

```diff
@@ -46,141 +35,126 @@ const logout = async (e) => {
  * This component is what allows the mainListItems feature to be displayed.
  * @returns {JSX.Element}
  */
-export const mainListItems = (
-  <div>
-    <Link className="SIDEBAR__link" to="/">
-      <ListItem button className="SIDEBAR__button">
-        <ListItemIcon>
-          <DashboardIcon className="SIDEBAR__icon" />
-        </ListItemIcon>
-        <ListItemText className="SIDEBAR__text" primary="Dashboard" />
-      </ListItem>
-    </Link>
+export function MainListItems() {
```

Comment:
refactored plain function 'mainListItems' into a component (so that I can use the useSelector to access the Store)